hermetic_build/library_generation/generate_library.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -eo pipefail
+set -exo pipefail
```

Comment:
`xtrace` was the only way I could debug my way out in local development. This decision may have taken longer for someone less familiar with the codebase.

Reply:
I didn't add `-x` because I think the extra logs may substantial and this level of logs is not recommended for production.

Do you think it's necessary to add these logs?

Reply:
I reverted this and explained how this would be useful in the dev guide.